Formula/literate-git.rb

```diff
@@ -0,0 +1,79 @@
+class LiterateGit < Formula
+  include Language::Python::Virtualenv
+
+  desc "Render hierarchical git repositories into HTML"
+  homepage "https://github.com/bennorth/literate-git"
+  url "https://github.com/bennorth/literate-git/archive/v0.3.1.tar.gz"
+  sha256 "f1dec77584236a5ab2bcee9169e16b5d976e83cd53d279512136bdc90b04940a"
+  head "https://github.com/bennorth/literate-git.git"
```

Comment:
Gah, sorry, one more thing: Why did you add `head`? Have you tested this locally? My suspicion is that this will fail and should be removed unless you have a good reason (like upstream adding nightly testing via `brew install --HEAD literate-git`).

Reply:
Wasn’t there for any particular reason, happy to remove it!